pages/login.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-use-before-define */
+import React, { useState } from 'react';
+import { useRouter } from 'next/router';
+import { Button, Form } from 'react-bootstrap';
+import { useAuth } from '../context/AuthContext';
+
+const Login = () => {
+  const router = useRouter();
+  //  const { user, login } = useAuth();
+  const { login } = useAuth();
+  const [data, setData] = useState({
+    email: '',
+    password: '',
+  });
+
+  const handleLogin = async (e: any) => {
+    e.preventDefault();
+
+    // console.log(user);
+    try {
+      await login(data.email, data.password);
+      router.push('/uploader');
```

Comment:
for now this is perfect but could be interesting to bring the user back to the page where they actually attempted to login from ie the page that directed them to the login screen

Reply:
We can discuss this further like you said